tools/ReplayAnalysis/api/app.py

```diff
@@ -50,11 +50,11 @@ def request_s3_data(filename):
     df = pd.DataFrame()
 
     for replay in selected_replays:
-        temp = None  # temporary df for individual replay
+        parsed = bucket_dict(replay["workload"])
         try:
             response = s3_client.get_object(
-                Bucket=replay["bucket"],
-                Key=f"analysis/{replay['replay_id']}/raw_data/{filename}",
+                Bucket=parsed.get("bucket_name"),
+                Key=f"{parsed.get('prefix')}{replay['replay_id']}/raw_data/{filename}",
```

Comment:
would this `parsed.get('prefix')` include a backslash at the end?

Reply:
Yup, see [this](https://github.com/aws/redshift-test-drive/pull/93/files#diff-b529ebd699e51252fae2773241e4cb2e818c6eaa5881cebbaa0c318351fafd76R32)